controllers/tutorial.controller.js

```
@@ -0,0 +1,74 @@
+/* eslint-disable no-console */
+const { response } = require('express');
+const { Tutorial } = require('../models/tutorial.schema');
+const mangoose = require('../init/db');
+
+// create Document or insert
+const createTutorial = async (req, res) => {
+  try {
+    const demoTutorial = new Tutorial({
+      id: "1",
+      published: true,
+    });
+    const result = await demoTutorial.save();
+    // eslint-disable-next-line no-console
+    console.log(result);
+  } catch (err) {
+    // eslint-disable-next-line no-console
+    console.log(err);
```

We should return a proper error response.